CHANGES.md

```diff
@@ -17,6 +17,8 @@ Format of the entries must be.
 
 ### Notable changes
 
+* Fix a bug in service.lua which was causing some wrong metrics labels for Admin Router and affecting request buffering by Admin Router. (DCOS_OSS-4950, DCOS_OSS-4999)
+
```

Comment:
Let's put this into the `Fixed and improved` section. I believe from 1.12's point of view the reference to metrics labels does not apply.